docs/evaluation/how_to_guides/evaluation/dataset_subset.mdx

```diff
@@ -0,0 +1,83 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+} from "@site/src/components/InstructionsWithCode";
+
+# How to evaluate on a split / filtered view of a dataset
+
+:::tip Recommended reading
+
+Before diving into this content, it might be helpful to read:
+
+- [guide on fetching examples](../datasets/manage_datasets_programmatically#fetch-examples).
+- [guide on creating/managing dataset splits](../datasets/manage_datasets_in_application#create-and-manage-dataset-splits)
+
+:::
+
+# How to: Evaluate on a filtered view of a dataset
+
+You can use the `list_examples` / `listExamples` method to fetch a subset of examples from a dataset to evaluate on. You can refer to guide above to learn more about the different ways to fetch examples.
+
+One common workflow is to fetch examples that have a certain metadata key-value pair.
```

Comment:
would comment that we have much more powerful filtering capabilities (particularly for metadata) using the filter parameter: https://docs.smith.langchain.com/evaluation/how_to_guides/datasets/manage_datasets_programmatically#list-examples-by-structured-filter